Checkers/Checkers.cs

```diff
@@ -6,70 +6,315 @@ namespace Checkers
 {
     class Program
     {
-        static void Main(string[] args)
+
+        static void Main(String[] args)
         {
-            Console.WriteLine("Hello World!");
+            Game game = new Game();
+            game.StartGame();
         }
     }
 
+    // Checker class
     public class Checker
     {
-        public string Symbol  { get; set; }
-        public int[] Position  { get; set; }
-        public string Color { get; set; }
-        
-        public Checker(string color, int[] position)
+        // Gets the Open or Closed circle value based on the checker's color
+        public String Symbol
+        {
+            get
+            {
+                if (Color == "black")
+                {
+                    int openCircleID = int.Parse("25CB", System.Globalization.NumberStyles.HexNumber);
+                    String openCircle = char.ConvertFromUtf32(openCircleID);
+
+                    return openCircle;
+                }
+                else if (Color == "white")
+                {
+                    int closedCircleID = int.Parse("25CF", System.Globalization.NumberStyles.HexNumber);
+                    String closedCircle = char.ConvertFromUtf32(closedCircleID);
+
+                    return closedCircle;
+                }
+                else
+                {
+                    return "";
+                }
+            }
+        }
+
+        public String Color { get; set; }
```

Comment:
Are you expecting the color of the checker to change? if not, then set could have been private